### PR TITLE
Add frame info to 'Extra' payload

### PR DESCRIPTION
**Pull request by vanjacosic**
@bendemboski reported that while opbeat-js does pick up the column number of each frame, it does not get shown anywhere in the Opbeat.com interface.

The column number is really important if you want to track down an issue in a minified file as the line number won't tell you very much (if anything at all).

So while we at Opbeat currently can't prioritize adding this to the interface at the moment, I decided to do a workaround in this library that gets the job done in a hackish way.

Here's how it works:

When a frame is captured, information about it (filename, line number, column and function) now get added to a new field in the optional `extra` option and sent to Opbeat.com. This means that it won't show up in the stacktrace, but if you switch to the Extras tab, you will be able to see all this info for each frame.
It ain't beautiful, but it gets the job done without having to change anything on the product side for now.

Screenshot:
screenshot

Ben, do you mind trying this out to see if it solves your problem for now? :) 

If it does, I can merge this PR as I think it could benefit other users of this library as well.

### CODE REVIEW

**Comment by mfc-julius** [comment]
Will it be merged?

As I said earlier, this is working and could help.

**Comment by vanjacosic** [comment]
@mfc-julius I was just waiting to see if @bendemboski would chime in with his thoughts.

But if it works for you, I'll go ahead and merge it. Thanks for feedback!

Currently the only issue with it is, that the data gets shown as a plain object and can be hard to parse (as you also mentioned in issue #5. This is currently a limitation on the Opbeat product side, we have added a ticket for it. So once we get to fixing that, the data will be displayed as a table (like HTTP data does in the HTTP tab) and should make this even better.

Merging :rocket:

**Comment by bendemboski** [comment]
Sorry, I have been on vacation and haven't had a chance to try it out, but
I'm eager to :)

B

**Comment by vanjacosic** [comment]
@bendemboski no worries - didn't mean to rush you! 

But since Julius said that it worked, I thought I'd just go ahead and merge it :) 

Feel free to get in touch via the support chat again and let me know how it works for you and if you have any trouble :+1:

**Comment by mfc-julius** [comment]
It's on our live site now with minified code.

Thanks for the merge, I could reach it via npm much easier.

Unfortunately, no errors occurred so far on our side, but will let you know if gonna have trouble finding the spot when it happens.
